README.md

```diff
@@ -6,73 +6,79 @@ https://firecloud.dsde-dev.broadinstitute.org/
 
 https://firecloud.dsde-staging.broadinstitute.org/
 
-## Getting Started
+## Technologies
 
-This project runs in a Docker container. Docker is available here:
+[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
 
-https://www.docker.com/
+We use the [Leiningen](http://leiningen.org/) build tool to manage ClojureScript dependencies.
 
-[ClojureScript](https://github.com/clojure/clojurescript) is used for the UI.
-We use the [Leiningen](http://leiningen.org/) build tool.
 The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
 a ClojureScript wrapper for [React](https://facebook.github.io/react/).
 
-For authentication, you will need to create a Google web application client ID from here:
+Figwheel replaces the running JavaScript within the page so changes are visible without a browser reload. More information [here](https://github.com/bhauman/lein-figwheel). [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
+
+## Getting Started
+
+Start with a [docker](https://www.docker.com/) environment.
+
+You will need to create a Google web application client ID from here:
 
 https://console.developers.google.com/
 
-## Building
+You can create credentials for yourself in **"APIs & Auth" -> "Credentials."** You want an **OAuth 2.0 client ID** with a **Web application** application type.
 
-Build once:
+Add your docker host as an authorized JavaScript origin. By convention, we use `dhost` as the hostname, so we add the following origins:
+- https://dhost
+- http://dhost
+
+The HTTP (vs. HTTPS) origin is necessary for hot-reloading during local development since Figwheel does not support HTTPS.
+
+To use this host, you'll need to add your docker machine's IP address (the address returned by `docker-machine ip default`) to your `/etc/hosts` file, e.g.,
 ```
-./script/dev/build-once.sh
+192.168.99.100 dhost
 ```
 
-Watch files and rebuild whenever a file is saved:
+Set your client ID in your environment:
 ```
-./script/dev/start-auto-build.sh
+export GOOGLE_CLIENT_ID='...'
 ```
 
-Watch files, rebuild, and reload changes into the running browser window:
+Start the server in docker:
 ```
-./script/dev/start-hot-reloader.sh
+./script/dev/start-server.sh
```

Comment:
1) Done.
2) Try running it outside your home directory. You should see an error message.
3) You shouldn't have to build the image. The one built by Jenkins should suffice (unless you are modifying the Dockerfile or testing modifications like for this PR)